app/screens/apps_form/apps_form_field.tsx

```diff
@@ -64,6 +65,7 @@ function selectDataSource(fieldType: string): string {
     }
 }
 
+const EMPTY_MENTION_KEYS: UserMentionKey[] = [];
```

Comment:
I'm seeing this defined in multiple places, let's move to the constants file

Reply:
After thinking about it for a while, I decided to remove the empty list in all the instances.

There is only one difference between passing and empty list and undefined, and it comes when rendering the `at mention`. When rendering the at mention, if an empty list is passed, no highlight will be done to the mention whatsoever. If undefined is passed, it will highlight the at mention if it is for the current user.

I feel that is the intended experience, instead of masking mentions to one in places like permalink preview, or slack attachment titles.